CMakeLists.txt

```diff
@@ -32,6 +32,12 @@ add_library(project_warnings INTERFACE) # Link this 'library' to use the warning
 include(cmake/CompilerWarnings.cmake)
 set_project_warnings(project_warnings)
 
+option(USE_DWARF4 "Use DWARF-4 instead of DWARF-5 for better LLDB compatibility" OFF)
+if(USE_DWARF4)
+  # LLDB doesn't seem to like bleeding edge gcc debug info. Enable this if your LLDB isn't showing source.
```

Comment:
Maybe specify the LLDB version (18?), so this can be reevaluated with the next releases, to see if the issue still exists.

Reply:
LLDB trunk is affected as well (v20), I wouldn't get my hopes up.
Unless we bisect compiler invocation flags until finding the culprit, which is a lot of work for little gain

Reply:
added your comment
and moved it before Dependencies.cmake, so we can debug into the dependencies as well